src/components/ContentNode/TabNavigator.vue

```diff
@@ -70,7 +70,12 @@ export default {
 @import 'docc-render/styles/_core.scss';
 
 .TabNavigator {
-  margin-bottom: $stacked-margin-large;
+  @include anything-before {
```

Comment:
The naming convention for these mixins is confusing because the "before" one actually applies the styles to the current element (when preceded by anything) while the "after" one applies to any element that follows this one.

Also, if we are going to create mixins for these, maybe it makes sense to have one mixin that applies the same spacing to both scenarios without having to duplicate it?

Reply:
Yep I saw that but it got lost in my mind lol. Will clean up.

Reply:
changed.